### PR TITLE
Fix #7091 #7092 #7113 - String updates for Widgets, Default Browser and ETP changes

### LINKED ISSUE #7091: FXIOS-699 ⁃ Land strings for iOS 14 Widgets
**Acceptance Criteria**

* All strings for the MVP iOS 14 widgets have landed in our repo
  * #7073 - Medium: Quick Actions
  * #7064 - Small: Quick Action (Regular Search, Private Search, Close Private Tabs, Go To Copied Link)  
  * #7067 - Medium: Top Sites
  * #7069 - Medium & Large: Quick View
  * #7068 - Pocket
* L10n team is informed of the new widgets for localization

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-699)

### PR DESCRIPTION
**Pull request by nbhasin2**

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -812,10 +814,76 @@ extension String {
...
+    public static let ViewMore = NSLocalizedString("TodayWidget.ViewMore", tableName: "Today", value: "View More", comment: "View More for Quick View widget in Gallery View where we don't know how many tabs might be opened")
+    
+    // Quick View Open Tabs - Large Size - Gallery View
+    public static let QuickViewLargeGalleryDescription = NSLocalizedString("TodayWidget.QuickViewLargeGalleryDescription", tableName: "Today", value: "Add shortcuts to your open tabs.", comment: "Description for Quick View widget in Gallery View where user can add it to home screen")
+
+    // Pocket - Large - Medium Size Widget
+    public static let PocketWidgetSubLabel = NSLocalizedString("TodayWidget.PocketWidgetSubLabel", tableName: "Today", value: "Firefox - Recommended by Pocket", comment: "Sub label for medium size Firefox Pocket stories widge widget. Pocket is the name of another app.")
+    public static let ViewMoreDots = NSLocalizedString("TodayWidget.ViewMoreDots", tableName: "Today", value: "View More...", comment: "View More... for Firefox Pocket stories widget where we don't know how many articles are available.")
```
Comment (garvankeeley) [inline Client/Frontend/Strings.swift:877]: `...` should be ellipsis char `…` ?

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -812,10 +814,76 @@ extension String {
...
+    
+    // Top Sites - Medium Size - Gallery View
+    public static let TopSitesGalleryTitle = NSLocalizedString("TodayWidget.TopSitesGalleryTitle", tableName: "Today", value: "Top Sites", comment: "Title for top sites widget to add Firefox top sites shotcuts to home screen")
+    public static let TopSitesGalleryDescription = NSLocalizedString("TodayWidget.TopSitesGalleryDescription", tableName: "Today", value: "Add shortcuts to frequently and recently visited sites.", comment: "Description for top sites widget to add Firefox top sites shotcuts to home screen")
+
+    // Quick View Open Tabs - Medium Size Widget
+    public static let QuickViewOpenTabsSubLabel = NSLocalizedString("TodayWidget.QuickViewOpenTabsSubLabel", tableName: "Today", value: "Firefox - Open Tabs", comment: "Sub label for Top Sites widget")
+    public static let MoreTabsLabel = NSLocalizedString("TodayWidget.MoreTabsLabel", tableName: "Today", value: "+%d More...", comment: "%d represents number and it becomes something like +5 more where 5 is the number of open tabs in tab tray beyond what is displayed in the widget")
```
Comment (garvankeeley) [inline Client/Frontend/Strings.swift:863]: `…` for elipsis char ?

Comment (nbhasin2) [reply Client/Frontend/Strings.swift]: Ah thanks for … char

**Comment by kaylagalway** [comment]
@Delphine Just wanted to ping you on this so that you can see the upcoming strings before we do an l10n string submission. If you have the time, definitely let us know if you see any glaring issues with any of the strings here!

**Pull request merged**